.github/workflows/check-links.yml

```diff
@@ -11,6 +11,7 @@ env:
 jobs:
   check:
     runs-on: ubuntu-latest
+    timeout-minutes: 10
```

Comment:
I think this should be longer it looks like sometimes it takes closer to 11 minutes
https://github.com/Kong/docs.konghq.com/actions/runs/7791282104
https://github.com/Kong/docs.konghq.com/actions/runs/7795979570

Reply:
This is the timeout for the check label job